### ISSUE TITLE
Format using user culture

### ISSUE THREAD

**Issue by hahn-kev** [sillsdev/languageforge-lexbox#647]
Currently we format dates using the culture of the selected language. So if the language is French it will be formatted according to the French date format, this is great. However For English we currently format the date using the US Culture: MM/DD/YY, not DD/MM/YY. Could we handle this better without having multiple English options? Maybe when we format the date we use the browser culture. The problem that might introduce is that the date doesn't match the language and instead matches the users browser settings.

Options:

1. Leave it as is, Brits and other English speakers get US dates
2. Add multiple English options. Not sure if that would require us to duplicate translation files or not.
3. Always use browser culture to format dates, could cause a mismatch between date format and language selected.
4. Use browser culture for English only

**Comment by myieye** [comment]
Oops. I feel like this is totally an i18n n00b mistake 🙈.
It would be best to save a user's whole locale.
I believe our i18n sependency uses tag shortening (I can't remember if that's the term that Martin used at the conference 😆). I.e. If you ask it to use 'en-UK' it will first look for a translation in en-uk.json then and if it doesn't find it it will cut off the '-uk' and look for the translation in 'en'.

In other words, even if we wanted to provide some specific translations in British English we wouldn't need to duplicate everything and I expect that date, number, currency etc. formatting would also just work.

**Comment by myieye** [comment]
I went with option 5 😉
5. Use browser locale if it matches the user's persisted locale, but is more specific

**Comment by megahirt** [comment]
option 5 sounds reasonable to me

**Comment by rmunn** [comment]
Dates should always be formatted YYYY-MM-DD anyway, no matter what the user prefers. :stuck_out_tongue_winking_eye:
